ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -203,10 +208,23 @@
        context
        (ingest-events/provider-autocomplete-suggestion-reindexing-event provider)))))
 
+(defn trigger-bulk-granule-task-cleanup
+  [context]
+  (let [providers (map :provider-id (mdb/get-providers context))]
```

Comment:
We should just delete all expired granule bulk update tasks at once. There is no need to clean up expired tasks by each provider.

Reply:
I was just thinking of that!